handlers/session_sync_handler.go

```diff
@@ -39,30 +29,20 @@ func (h *Handler) SessionSyncHandler(w http.ResponseWriter, req *http.Request, _
 	}
 	logrus.Debugf("final list of active adapters: %+v", meshAdapters)
 	prefObj.MeshAdapters = meshAdapters
-	err = provider.RecordPreferences(req, user.UserID, prefObj)
+	err := provider.RecordPreferences(req, user.UserID, prefObj)
 	if err != nil { // ignoring errors in this context
 		logrus.Errorf("unable to save session: %v", err)
-		// http.Error(w, "unable to save session", http.StatusInternalServerError)
-		// return
 	}
 
 	if prefObj.K8SConfig != nil {
 		if prefObj.K8SConfig.ServerVersion == "" {
 			// fetching server version, if it has not already been
 			prefObj.K8SConfig.ServerVersion, _ = helpers.FetchKubernetesVersion(prefObj.K8SConfig.Config, prefObj.K8SConfig.ContextName)
-			// if err != nil {
-			// 	http.Error(w, "unable to ping the kubernetes server", http.StatusInternalServerError)
-			// 	return
-			// }
 		}
 
 		if len(prefObj.K8SConfig.Nodes) == 0 {
 			// fetching nodes, if it has not already been
 			prefObj.K8SConfig.Nodes, _ = helpers.FetchKubernetesNodes(prefObj.K8SConfig.Config, prefObj.K8SConfig.ContextName)
-			// if err != nil {
-			// 	http.Error(w, "unable to fetch nodes metadata from the kubernetes server", http.StatusInternalServerError)
-			// 	return
-			// }
```

Comment:
Should we uncomment instead of deleting? Good to capture this error?

Reply:
Error is already captured above. There is no new error assignment for this block, hence this block is redundant.